grails-app/migrations/views/cycle-count-session.sql

```diff
@@ -41,42 +41,62 @@ SELECT
        AND pai.location_id = product_availability.location_id)                       as negative_item_count,
 
     -- Date last counted
+    -- NULL as date_last_count,
     (select max(date_counted)
      from product_count_history
      where product_count_history.product_id = product_availability.product_id
        and product_count_history.inventory_id = location.inventory_id
      group by inventory_id, product_id)                                              as date_last_count,
 
-    NULL                                                                             as date_next_count,
+#    NULL as date_next_count,
+#    NULL as days_until_next_count,
```

Comment:
I'm still going back and forth with these NULL columns as I work through the performance issues, but will remove them before we merge.